common/dynamicconfig/constants.go

```diff
@@ -1198,6 +1198,11 @@ Set to zero to disable proactive unload.`,
 		`MatchingQueryWorkflowTaskTimeoutLogRate defines the sampling rate for logs when a query workflow task times out. Since
 these log lines can be noisy, we want to be able to turn on and sample selectively for each affected namespace.`,
 	)
+	CachedPhysicalInfoByBuildIdTTL = NewTaskQueueDurationSetting(
+		"matching.cachedPhysicalInfoByBuildIdTTL",
+		time.Second,
+		`cachedPhysicalInfoByBuildIdTTL acts as a cache TTL for cachedPhysicalInfoByBuildIdCache`,
+	)
```

Comment:
```suggestion
	PhysicalTaskQueueInfoByBuildIdTTL = NewTaskQueueDurationSetting(
		"matching.physicalTaskQueueInfoByBuildIdTTL",
		time.Second,
		`physicalTaskQueueInfoByBuildIdTTL is the cache TTL for physicalTaskQueueInfoByBuildIdCache`, which is used in DescribeTaskQueue,
	)
```

Reply:
I like this naming better because:
- I think it's important to get the term "task queue" in there, so that non-matching-crew folks know this is related to task queue info and describe task queue
- "cached...Cache" is a bit redundant, so I think you can drop the "cached" from the front :)

Reply:
yeah, good pointer I agree.